lib/Conversion/TorchToLinalg/TorchToLinalg.cpp

```diff
@@ -634,6 +693,26 @@ class ConvertAtenBatchNormOp : public OpConversionPattern<AtenBatchNormOp> {
     auto runningMeanType = runningMean.getType().cast<RankedTensorType>();
     auto runningVarType = runningVar.getType().cast<RankedTensorType>();
 
+    // Calculate the inverted STD.
```

Comment:
The extra loop to calculate inverted STD is less efficient for BatchNorm than before. Can you introduce a `createLinalgPayloadCalculationForNormOpsWithVar` to take `var` and another `createLinalgPayloadCalculationForNormOpsWithRstd` to take `rstd`
```
createLinalgPayloadCalculationForNormOpsWithVar(....Value var) {
    Value truncatedEps = b.create<arith::TruncFOp>(loc, elemTy, eps);
    Value varPlusEps = b.create<arith::AddFOp>(loc, var, truncatedEps);
    Value rSTD = b.create<math::RsqrtOp>(loc, varPlusEps);
    createLinalgPayloadCalculationForNormOpsWithRstd(... rSTD);
}
```

Reply:
Added this.